Cargo.toml

```diff
@@ -46,11 +48,13 @@ serde = "1.0"
 serde_derive = "1.0"
 url = "2.2.2"
 reqwest = { version = "0.11", features = ["blocking", "rustls-tls", "rustls-tls-native-roots"], optional = true, default-features = false }
-picky = { version = "7.0.0-rc.4", default-features = false }
-picky-krb = "0.6.0"
-picky-asn1 = { version = "0.7.1", features = ["chrono_conversion"] }
-picky-asn1-der = "0.4.0"
-picky-asn1-x509 = { version = "0.9.0", features = ["pkcs7"] }
+
+picky = { git = "https://github.com/Devolutions/picky-rs.git", rev = "fda2fad12a704f2d5008f1ec4df62e5ca8a5e8ff", default-features = false }
+picky-asn1 = { git = "https://github.com/Devolutions/picky-rs.git", rev = "fda2fad12a704f2d5008f1ec4df62e5ca8a5e8ff", features = ["chrono_conversion"] }
+picky-asn1-der = { git = "https://github.com/Devolutions/picky-rs.git", rev = "fda2fad12a704f2d5008f1ec4df62e5ca8a5e8ff" }
+picky-asn1-x509 = { git = "https://github.com/Devolutions/picky-rs.git", rev = "fda2fad12a704f2d5008f1ec4df62e5ca8a5e8ff", features = ["pkcs7"] }
+picky-krb = { git = "https://github.com/Devolutions/picky-rs.git", rev = "fda2fad12a704f2d5008f1ec4df62e5ca8a5e8ff" }
```

Comment:
I will update the `picky-*` dependencies once https://github.com/Devolutions/picky-rs/pull/233 is merged and the new versions of the crates are published

Reply:
Sounds good. Newer picky versions will be published in the next few days, at least before Friday :+1: (waiting for a pull request to get merged)

Reply:
I have two more pull requests in the scope of the Kerberos smart card logon. Maybe we can merge this PR as it is? Anyway, we merge it in the dev branch and I can fix it later. I just want to minimize the code review time

Reply:
It won’t be necessary, new picky versions are incoming in the next few hours

Reply:
Sorry for the delay. The new versions are available!

Reply:
I updated `picky-*` dependencies